src/main/java/frc/robot/RobotContainer.java

```diff
@@ -431,28 +431,28 @@ private Command shortIntakeForTime(Intake intake, Indexer indexer) {
 
         private Command setIndexerAndIntakeSpeed(Indexer indexer, Intake intake, double speed) {
                 return new ParallelCommandGroup(
-                                new RunCommand(() -> intake.setMotor(speed), m_intake),
-                                new RunCommand(() -> indexer.setMotor(speed), m_indexer)).withTimeout(0.4);
+                                new RunCommand(() -> intake.setMotor(speed), intake),
+                                new RunCommand(() -> indexer.setMotor(speed), indexer)).withTimeout(0.4);
         }
 
         // waiting 0.5 seconds to get shooter up to speed
         private Command shootWhenUpToSpeed() {
                 return new SequentialCommandGroup(
                                 new ParallelCommandGroup(
                                                 new SequentialCommandGroup(
-                                                                new WaitUntilCommand(() -> m_shooter
-                                                                                .getEncoderSpeed() >= (m_arm
+                                                                new WaitUntilCommand(() -> shooter
+                                                                                .getEncoderSpeed() >= (arm
                                                                                                 .getSpeedFromArmHeight()
-                                                                                                * Constants.ShooterConstants.SHOOT_MAX_SPEED_RPS)),
-                                                                new RunCommand(() -> m_indexer.setMotor(
+                                                                                                * ShooterConstants.SHOOT_MAX_SPEED_RPS)),
+                                                                new RunCommand(() -> indexer.setMotor(
                                                                                 Constants.IndexerConstants.INDEXER_IN_SPEED),
-                                                                                m_indexer)),
-                                                new RunCommand(() -> m_shooter.setMotor(m_arm.getSpeedFromArmHeight()),
-                                                                m_shooter))
+                                                                                indexer)),
+                                                new RunCommand(() -> shooter.setMotor(arm.getSpeedFromArmHeight()),
```

Comment:
again, the shooter command should be coming from the command factory in the subsystem